kubernetes-repositories.md

```diff
@@ -0,0 +1,106 @@
+## Kubernetes Repositories
+
+
+This document attempts to describe a process for creating and associating github repositories with the Kubernetes project.
+
+The document presents a tiered system of repositories with increasingly strict requirements in an attempt to provide the right level of oversight and flexibility for a variety of different projects.
+
+
+### Associated Repositories
+
+Associated repositories conform to the Kubernetes community standards for a repository, but otherwise have no restrictions. Associated repositories exist solely for the purpose of making it easier for the Kubernetes community to work together. There is no implication of support or endorsement of any kind by the Kubernetes project, the goals are purely logistical. 
+
+#### Goals
+
+To facilitate contributions and collaboration from the broader Kubernetes community.  Contributions to random projects with random CLAs (or DCOs) can be logistically difficult, so associated repositories should be easier.
+
+
+#### Rules
+
+   * Must adopt the Kubernetes Code of Conduct statement in their repo.
+   * All code projects use the Apache License version 2.0. Documentation repositories should use the Creative Commons License version 4.0.
+   * Must adopt the CNCF CLA bot automation for pull requests.
+
+
+### SIG repositories
+
+SIG repositories serve as temporary homes for SIG-sponsored experimental projects or prototypes of new core functionality, or as permanent homes for SIG-specific tools.
+
+#### Goals
+
+To provide a place for SIGs to collaborate on projects endorsed by and actively worked on by members of the SIG. SIGs should be able to approve and create new repositories for SIG-sponsored projects without requiring higher level approval from a central body (e.g. steering committee or sig-architecture)
+
+#### Rules
```

Comment:
Do we want to have guidelines to setup some common teams for those repos? E.g.:
- \<signame>-\<project>-reviewers
- \<signame>-\<project>-admins
- \<signame>-\<project>-bots

I think the bots may be useful, I like the reviewers team for tagging them on PRs to ask for, well, reviews, ... I am not sure if there should be more teams.

Reply:
I will do a separate doc with concrete process.

Reply:
Teams are covered by:
https://github.com/kubernetes/community/blob/master/org-owners-guide.md